lab_3_generate_by_ngrams/main.py

```diff
@@ -208,6 +302,17 @@ def build(self) -> int:
         In case of corrupt input arguments or methods used return None,
         1 is returned
         """
+        if not isinstance(self._encoded_corpus, tuple) or len(self._encoded_corpus) == 0:
+            return 1
+        n_grams = self._extract_n_grams(self._encoded_corpus)
+
+        if not isinstance(n_grams, tuple) or len(n_grams) == 0:
+            return 1
+
+        for n_gram in n_grams:
+            same_n_grams = [same for same in n_grams if same[:-1] == n_gram[:-1]]
```

Comment:
count?

Reply:
either i don't understand or i can't imagine why we need apply this with adding extra strings(

Reply:
I think, you should replace this with count